src/tools/cephfs/cephfs_shell.py

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env python3
+# coding=utf-8
+
+import argparse
+import os
+import sys
+import cmd
+import cephfs as libcephfs
+import shlex
+import shutil
+
+cephfs =  None
+
+def setup_cephfs():
+    """ Mouting a cephfs """
+    global cephfs
+    cephfs = libcephfs.LibCephFS(conffile='/home/admin1/Documents/ceph/build/ceph.conf')
+    cephfs.mount()
+
+class CephfsShell(cmd.Cmd):
+    def __init__(self):
+        super().__init__()
+        self._set_prompt()
+        self.intro = 'Ceph File System Shell'
+
+    def _set_prompt(self):
+        self.cwd = os.getcwd()
+        self.prompt = '\033[01;33mCephFS >>>\033[00m ' #self.colorize('{!r} $ '.format(self.cwd), 'cyan')
+
+    def postcmd(self, stop, line):
+        self._set_prompt()
+        return stop
+
+    def do_mkdir(self,arglist):
+        """Create directory.
+            Usage: mkdir <new_dir> <permission> """    
+        parser = argparse.ArgumentParser(description='Create Directory.')
```

Comment:
We won't want to recreate the parser everytime a shell command is run. It would be a good idea to pull this `parser` variable out into a static class member. You can read more about this here:

http://radek.io/2011/07/21/static-variables-and-methods-in-python/
https://stackoverflow.com/questions/872973/pythonic-way-to-initialize-complex-static-data-members

Using a static method to create the parser (one method for each command) seems reasonable to me. Then call the static method to initialize the parser for the command, named `mkdir_parser`. (There may be more "Pythonic" ways to solve this though.)